command/push.py

```diff
@@ -1,8 +1,35 @@
+import base64
+import os
+import requests
+import tarfile
+
 class PushCmd():
     name = 'push'
 
     def __init__(self):
         pass
         
-    def push(self):
-        print("Hello, world. The push feature is currently under construction.")
+    def push(self, bundle_dir, namespace, repository, release, auth_token):
+        base64_bundle = self.__create_base64_bundle(bundle_dir, repository)
+        self.__push_to_registry(namespace, repository, release, base64_bundle, auth_token)
+
+    def __create_base64_bundle(self, bundle_dir, repository):
```

Comment:
This method accepts a bundle directory. Marketplace expects a single bundle to exist within the tar pulled from Quay.

Reply:
Understood. LGTM. :+1: